superset-frontend/src/common/components/Radio/Radio.stories.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { useArgs } from '@storybook/client-api';
+import { Radio } from './index';
+
+export default {
+  title: 'Radio',
+  component: Radio,
+  argTypes: {
+    theme: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+};
+
+export const SupersetRadio = (_: any) => {
```

Comment:
Could this not just be like so:
```suggestion
export const SupersetRadio = () => {
```

Reply:
@villebro If I remove this parameter Storybook will disable the controls panel. I think it depends on having parameters.

Reply:
@villebro I was able to resolve this using a combination of `argTypes` and `hideNoControlsWarning` 😄

Reply:
Oh god, sorry about that! 😄 Looks good, let me approve!